gp-templates/glossary-import.php

```diff
@@ -19,6 +19,14 @@
 		<label for="import-file"><?php _e( 'Import File:', 'glotpress' ); ?></label>
 		<input type="file" name="import-file" id="import-file" />
 	</p>
+	<?php if ( $can_write ) : ?>
+		<p>
+			<label for="import-replace">
+				<input type="checkbox" id="import-replace" name="import-replace" />
+				<?php _e( 'Replace existing glossary. Warning: all existing glossary entries will be deleted!' ); ?>
```

Comment:
I'm not sure "Replace" is the right terminology here, perhaps "Flush" or "Purge" would be a stronger term to make sure it's clear what's about to happen.